policyengine_us/variables/gov/states/nh/tax/income/exemptions/nh_base_exemption.py

```diff
@@ -0,0 +1,12 @@
+from policyengine_us.model_api import *
+
+
+class nh_base_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "New Hampshire base exemption household level"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.NH
+
+    adds = "gov.states.nh.tax.income.exemptions.amount.base"
```

Comment:
`adds` has to be a list, which is why the test is failing. I filed https://github.com/PolicyEngine/policyengine-core/issues/102 to support this use case.

In the meantime I'd suggest using a `formula` that returns the parameter value.